3-platform/shopping-cart/db/postgres/01-create-shopping-cart.sql

```diff
@@ -0,0 +1,16 @@
+-- DROPPING TABLE IF EXISTS
+DROP TABLE IF EXISTS shopping_cart;
+
+-- CREATING TABLE "Shopping cart"
+CREATE TABLE shopping_cart (
+                               cart_id VARCHAR(255) NOT NULL,
+                               product_ids TEXT[],
```

Comment:
Don't make a field like an array if it's not necessary. It would be complicated to manipulate that data.
Let it rename as product_id with varchar type (FK). Also, there will not be any PK.

Reply:
I’m not really sure that the index in this table will improve anything, but I fixed it too. )